### PR TITLE
refactor:NameWallet,CredentialOffer,onboardingpages.exportwalletConfi…

### PR DESCRIPTION
**Pull request by poonam-ghewande**
1. Changed onboarding  text content.
2. Added limit  for naming wallet.
3. Bold issuer name on credential offer .
4. Resolved decline popup issue.
5. Added random index export wallet confirmation.

### CODE REVIEW

File: app/screens/ExportWalletConfirmation.tsx [hunk 1]
```diff
@@ -128,18 +124,23 @@ function ExportWalletConfirmation() {
     },
   })
 
+  // useEffect(() => {
```
Comment (nbAmit) [inline app/screens/ExportWalletConfirmation.tsx:127]: remove code if not required

File: app/screens/CredentialOffer.tsx [hunk 2]
```diff
@@ -72,11 +72,14 @@ const CredentialOffer: React.FC<CredentialOfferProps> = ({ navigation, route })
     headerText: {
       ...ListItems.recordAttributeLabel,
       flexShrink: 1,
-      fontWeight: 'bold',
+      // fontWeight: 'bold',
```
Comment (nbAmit) [inline app/screens/CredentialOffer.tsx:75]: same here

File: app/screens/ExportWalletConfirmation.tsx [hunk 3]
```diff
@@ -323,8 +321,19 @@ function ExportWalletConfirmation() {
...
+              <View>
+                <Text style={[styles.rowAddItemText]}>{item}</Text>
+              </View>
+            </TouchableOpacity>
+          ))}
 
-          {parms?.params?.phraseData?.map((item: string[], index: number) => (
+          {/* {phraseData?.map((item, index) => (
             <TouchableOpacity
               disabled={matchPhrase}
```
Comment (nbAmit) [inline app/screens/ExportWalletConfirmation.tsx:338]: Same

File: app/screens/ExportWalletConfirmation.tsx (hunk 1 above)
Comment (poonam-ghewande) [reply app/screens/ExportWalletConfirmation.tsx]: removed

File: app/screens/CredentialOffer.tsx (hunk 2 above)
Comment (poonam-ghewande) [reply app/screens/CredentialOffer.tsx]: removed commented code

File: app/screens/ExportWalletConfirmation.tsx (hunk 3 above)
Comment (poonam-ghewande) [reply app/screens/ExportWalletConfirmation.tsx]: removed commented code

**Review by amitpadmani-awts — APPROVED** [review]

**Pull request merged**